packages/integration-tests/src/package_manager_sanity_checks.test.ts

```diff
@@ -69,7 +76,11 @@ void describe('getting started happy path', async () => {
   void it('creates new project and deploy them without an error', async () => {
     await runPackageManager(
       packageManager,
-      ['create', 'amplify', '--yes'],
+      [
+        'create',
+        `amplify${packageManager.startsWith('yarn') ? '' : '@beta'}`,
```

Comment:
`yarn create amplify@beta` doesn't work 😭  which is why we have to treat it differently. 


I think it's a bug of yarn because it does install correctly, but it calls a wrong binary. 

```
❯ yarn create amplify@0.6.1-beta.0 --yes
yarn create v1.22.21
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning "create-amplify > @aws-amplify/plugin-types@0.8.0" has unmet peer dependency "aws-cdk-lib@^2.110.1".
warning "create-amplify > @aws-amplify/plugin-types@0.8.0" has unmet peer dependency "constructs@^10.0.0".
[4/4] 🔨  Building fresh packages...

success Installed "create-amplify@0.6.1-beta.0" with binaries:
      - create-amplify
/bin/sh: /Users/xxxxxxxx/.yarn/bin/create-amplify@0.6.1-beta.0: No such file or directory
error Command failed.
Exit code: 127
Command: /Users/xxxxxxxx/.yarn/bin/create-amplify@0.6.1-beta.0
Arguments: --yes
Directory: /Volumes/workplace/playground/amplify-2024-02-12
Output:

info Visit https://yarnpkg.com/en/docs/cli/create for documentation about this command.
```

So I think, a workaround would be -- if it's `yarn`, we install beta first, then `yarn create amplify`

Reply:
We should probably do the workaround. Otherwise we are effectively not running this test on yarn

Reply:
Done 52dc1f706b